README.md

```diff
@@ -28,116 +23,22 @@ When the reader has completed this Code Pattern, they will understand how to:
-
-### 1. Clone the repo
-
-Clone `watson-conversation-slots-intro` locally. In a terminal, run:
-
-  `$ git clone https://github.com/ibm/watson-conversation-slots-intro`
-
-We’ll be using the file [`data/watson-pizzeria.json`](data/watson-pizzeria.json) to upload
-the Assistant Intents, Entities, and Dialog Nodes.
-
-### 2. Create IBM Cloud services
 
-Create the following service and name it `wcsi-conversation-service`:
-
-  * [**Watson Assistant**](https://cloud.ibm.com/catalog/services/conversation)
-
-### 3. Configure Watson Assistant
-
-#### Import the Assistant workspace.json:
-
-* Find the Assistant service in your IBM Cloud Dashboard.
-* Click on the `Manage` tab and then click on `Launch Watson Assistant`.
-* Go to the `Skills` tab.
-* Click `Create skill`
-* Click the `Import skill` tab.
-* Click `Choose JSON file`, go to your cloned repo dir, and `Open` the workspace.json file in [`data/watson-pizzeria.json`](data/watson-pizzeria.json).
-* Select `Everything` and click `Import`.
-
-To find the `WORKSPACE_ID` for Watson Assistant:
-
-* Go back to the `Skills` tab.
-* Find the card for the workspace you would like to use. Look for `WatsonPizzeria`.
-* Click on the three dots in the upper right-hand corner of the card and select `View API Details`.
-* Copy the `Workspace ID` GUID.
-
-!["Get Workspace ID"](https://github.com/IBM/pattern-utils/blob/master/watson-assistant/assistantPostSkillGetID.gif)
-
-* In the next step, you will put this `Workspace ID` into the `.env file as ``WORKSPACE_ID``.
-
-### 4. Get IBM Cloud services credentials and add to .env file
-
-* Move the `watson-conversation-slots-intro/env.sample` file to ``watson-conversation-slots-intro/.env``.
-
-* Use the `apikey` from your Watson Assistant service credentials in the `.env` file. Note that older services might still use username/password, so comment those out and use instead, if applicable.
-
-!["Assistant Credentials"](https://github.com/IBM/pattern-utils/blob/master/watson-assistant/watson_assistant_api_key.png)
-
-* Put the `Workspace ID` into the `.env file as ``WORKSPACE_ID``.
-
-
-```bash
-WORKSPACE_ID=<put workspace id here>
-
-# Watson Assistant Authentication using IAM
-CONVERSATION_IAM_APIKEY=<put assistant IAM apikey here>
-CONVERSATION_URL=<put assistant url here>
-```
-
-### 5. Run the application
-
-#### If you used the Deploy to IBM Cloud button...
+* [Node.js](https://nodejs.org/): An asynchronous event driven JavaScript runtime, designed to build scalable applications.
 
-If you used ``Deploy to IBM Cloud``, the setup is automatic.
+## Watch the Video
 
-#### If you decided to run the app locally...
+| Using Cloud Foundry on IBM Cloud | Using IBM Cloud's Kubernetes service |
+| - | - |
+| [![video](https://i.ytimg.com/vi/dTuJU8-FnNM/0.jpg)](https://youtu.be/dTuJU8-FnNM) | [!["video"](https://i.ytimg.com/vi/G-rESweRG84/0.jpg)](https://youtu.be/G-rESweRG84) |
 
-```bash
-npm install
-npm start
-```
+## Deployment options
```

Comment:
How about we add some instruction here?
`Click on one of the deployment options below:`